src/Sarif/ExtensionMethods.cs

```diff
@@ -336,5 +344,105 @@ public static string GetFirstSentence(string text)
             }
             return text.TrimEnd('.') + ".";
         }
+
+        /// <summary>Retrieves a property value if it exists, or null.</summary>
+        /// <param name="properties">A properties object from which the property shall be
+        /// retrieved, or null.</param>
+        /// <param name="key">The property name / key.</param>
+        /// <returns>
+        /// If <paramref name="properties"/> is not null and an entry for the supplied key exists, the
+        /// value associated with that key; otherwise, null.
+        /// </returns>
+        internal static string PropertyValue(this Dictionary<string, string> properties, string key)
```

Comment:
I see that you combined two files, Extensions.cs and ExtensionMethods.cs. That's fine.

Are there _any other changes_ in the migrated code that I should look at?

Reply:
no

---
In reply to: [136150453](https://github.com/Microsoft/sarif-sdk/pull/612#discussion_r136150453) [](ancestors = 136150453)